### PR TITLE
Update test status to reflect NAPALM distributions that have release Python3 support

### PR DESCRIPTION
**Pull request by ktbyers**
Update test status to reflect NAPALM distributions that have release Python3 support

### CODE REVIEW

File: test/unit/TestGetNetworkDriver.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ class TestGetNetworkDriver(unittest.TestCase):
                            'nxos', 'pluribus', 'panos')
     elif PY3:
         # Drivers that support python2 and python3
-        network_drivers = ('ios',)
+        network_drivers = ('eos', 'ios', 'nxos')
```
Comment (mirceaulinic) [inline test/unit/TestGetNetworkDriver.py]: Also iosxr is py3 now, thanks to Patrick.

Comment (ktbyers) [inline test/unit/TestGetNetworkDriver.py]: Yes, iosxr is still only in 'develop' not on 'master' (at least as of today).

Comment (mirceaulinic) [inline test/unit/TestGetNetworkDriver.py]: Right, my fault. Forgot to release, apologise!

Comment (GGabriele) [inline test/unit/TestGetNetworkDriver.py]: panos is py3 as well now

Comment (ktbyers) [inline test/unit/TestGetNetworkDriver.py]: Awesome...will update to add ios-xr and panos to this PR.

Comment (mirceaulinic) [inline test/unit/TestGetNetworkDriver.py]: + junos :)

**Comment by ppieprzycki** [comment]
@ktbyers Please add to this list napalm-vyos it has python 3 checks  https://travis-ci.org/napalm-automation/napalm-vyos

**Comment by ktbyers** [comment]
@ppieprzycki Okay, done.

**Comment by ktbyers** [comment]
@ppieprzycki You don't have any releases:

https://github.com/napalm-automation/napalm-vyos/releases

So pip install napalm-vyos is failing:

```
$ pip install napalm-vyos
You are using pip version 7.0.3, however version 9.0.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
Collecting napalm-vyos
  Could not find a version that satisfies the requirement napalm-vyos (from versions: )
No matching distribution found for napalm-vyos
```

**Comment by dbarrosop** [comment]
> You don't have any releases:

@ktbyers I will fix that soon

I was waiting to have all the "administrative" tasks for a first release ready. I guess I will fix that today, probably before you read this message ;)

**Comment by dbarrosop** [comment]
It is fixed now : )

**Comment by ktbyers** [comment]
@dbarrosop @mirceaulinic Could one of you review/approve this. It should have all of the necessary changes.

**Comment by ktbyers** [comment]
@dbarrosop @mirceaulinic FYI, I added another change to handle an issue I was seeing with unit tests and the tests failing incorrectly due to a 'long' data type being returned.

**Comment by dbarrosop** [comment]
Approved and merged : )

**Pull request merged**